rfq/negotiator.go

```diff
@@ -23,6 +31,18 @@ type NegotiatorCfg struct {
 	// messages. These are messages which are destined to be sent to peers.
 	OutgoingMessages chan<- rfqmsg.OutgoingMsg
 
+	// AcceptPriceDeviationBp specifies the maximum allowable price
+	// deviation, in basis points. This parameter defines the maximum
+	// percentage deviation that the price returned by the price oracle
+	// service can have from a peer's quote accept price for this node to
+	// use the accepted quote.
+	AcceptPriceDeviationBp uint64
```

Comment:
Basis points is very commonly used in finance. Where as I would associate PPM with environmental science or chemistry. So I find LND's use of PPM instead of BP strange. I can see that it's used for the fee rate in LND.

As you suggest, I'll switch to PPM in this PR for consistency.